src/daemon/revaultd.rs

```diff
@@ -592,6 +592,11 @@ mod tests {
         path.push("../../test_data/invalid_config.toml");
         Config::from_file(Some(path.clone())).expect_err("Parsing invalid config file");
 
+        path.pop();
+        path.push("invalid_config_network.toml");
+        Config::from_file(Some(path.clone()))
+            .expect_err("Our bitcoin network is regtest but one xpub has network bitcoin");
```

Comment:
Wait, no the network in the config is signet?

Reply:
But somehow the test passes

Reply:
Daniela forgot the meaning of `expect_err` it seems 😭 

```
-        Config::from_file(Some(path.clone()))
-            .expect_err("Our bitcoin network is regtest but one xpub has network bitcoin");
+        assert!(Config::from_file(Some(path.clone()))
+            .unwrap_err()
+            .to_string()
+            .contains("Our bitcoin network is signet but one xpub has network bitcoin"));
```

Reply:
eh, so did i